docs/usage/types/list_types.md

```diff
@@ -1,60 +1,63 @@
 `list`
 : allows `list`, `tuple`, `set`, `frozenset`, `deque`, or generators and casts to a list;
-  see `typing.List` below for sub-type constraints
+  when a generic parameter is provided, the appropriate validation is applied to all items of the list
+
+`typing.List`
+: handled the same as `list` above
```

Comment:
I think the original ordering was reflective of the fact that the non-`typing` versions of the annotations couldn't support generic parameters. Now that they can, I think it makes sense to say less about the `typing`-specific ones.